[What is Displayed on Main Terminal]

PS C:\Users\gastb\Documents\kubernetes> .\run_kubernetes.sh
PS C:\Users\gastb\Documents\kubernetes> kubectl get pod
NAME         READY   STATUS              RESTARTS   AGE
predictapp   0/1     ContainerCreating   0          2m12s
PS C:\Users\gastb\Documents\kubernetes> kubectl get pod
NAME         READY   STATUS    RESTARTS   AGE
predictapp   1/1     Running   0          5m18s
PS C:\Users\gastb\Documents\kubernetes> .\run_kubernetes.sh
Error from server (AlreadyExists): pods "predictapp" already exists
NAME         READY   STATUS    RESTARTS   AGE
predictapp   1/1     Running   0          13m
Forwarding from 127.0.0.1:8000 -> 80
Forwarding from [::1]:8000 -> 80
Handling connection for 8000
Handling connection for 8000
Handling connection for 8000

[What is Outputted to Kubernetes predictapp log]

PS C:\Users\gastb\Documents\kubernetes> kubectl logs predictapp
 * Serving Flask app "app" (lazy loading)
 * Environment: production
   WARNING: Do not use the development server in a production environment.
   Use a production WSGI server instead.
 * Debug mode: on
 * Running on http://0.0.0.0:80/ (Press CTRL+C to quit)
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 623-034-185
[2020-08-05 18:36:49,995] INFO in app: JSON payload:
{'CHAS': {'0': 0}, 'RM': {'0': 6.575}, 'TAX': {'0': 296.0}, 'PTRATIO': {'0': 15.3}, 'B': {'0': 396.9}, 'LSTAT': {'0': 4.98}}
[2020-08-05 18:36:50,007] INFO in app: Inference payload DataFrame:
   CHAS     RM    TAX  PTRATIO      B  LSTAT
0     0  6.575  296.0     15.3  396.9   4.98
[2020-08-05 18:36:50,016] INFO in app: Scaling Payload:
   CHAS     RM    TAX  PTRATIO      B  LSTAT
0     0  6.575  296.0     15.3  396.9   4.98
[2020-08-05 18:36:50,019] INFO in app: output prediction: [20.35373177134412]
127.0.0.1 - - [05/Aug/2020 18:36:50] "POST /predict HTTP/1.1" 200 -
